cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1382,6 +1382,25 @@ def _check_spatial_obs(self):
         self._validate_spatial_tissue_position("array_row", 0, 77)
         self._validate_spatial_tissue_position("in_tissue", 0, 1)
 
+        self._validate_spatial_is_primary_data()
+
+    def _validate_spatial_is_primary_data(self):
+        """
+        Validate is_primary_data for spatial datasets.
+        """
+        is_single = (
```

Comment:
Could we use `_is_single()` here once this is rebased on `main`?

Reply:
ah yes good call